board_client-sim/main.py

```
@@ -0,0 +1,96 @@
+)
+args = parser.parse_args()
+
+# Patch sys.path so imports work
+import os
+sys.path.insert(0, os.path.abspath("."))
+
+# Set environment variable for model selection
+os.environ["UNICORN_SIM_MODEL"] = args.model
+
+# Import the simulation hardware layer
+from sim.hardware_sim import graphics, gu, set_brightness, WIDTH, HEIGHT, MODEL
```

Ruff (0.11.9)

`sim.hardware_sim.WIDTH` imported but unused

Remove unused import

(F401)